src/encoded/types/dataset.py

```diff
@@ -804,7 +804,9 @@ def control_type(self, request, related_datasets):
 class MatchedSet(Series):
     item_type = 'matched_set'
     schema = load_schema('encoded:schemas/matched_set.json')
-    embedded = Series.embedded
+    embedded = Series.embedded + [
```

Comment:
The data is small, so it won't make a big difference. But in general,  .extends() or something else my be better. '+' on list is very slow. See- https://stackoverflow.com/a/12088747/178550